From: gregkh@suse.de
Subject: [PATCH] USB: fix up HZ change in zd1201 driver.
ChangeSet 1.2020.1.155, 2005/03/08 01:05:49-08:00, gregkh@suse.de

[PATCH] USB: fix up HZ change in zd1201 driver.

Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>


 drivers/usb/net/zd1201.h |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)


diff -Nru a/drivers/usb/net/zd1201.h b/drivers/usb/net/zd1201.h
--- a/drivers/usb/net/zd1201.h	2005-03-08 16:39:11 -08:00
+++ b/drivers/usb/net/zd1201.h	2005-03-08 16:39:11 -08:00
@@ -65,9 +65,9 @@
 #define ZD1201SIWMAXASSOC SIOCIWFIRSTPRIV+4
 #define ZD1201GIWMAXASSOC ZD1201SIWMAXASSOC+1
 
-#define ZD1201_FW_TIMEOUT	(1*HZ)
+#define ZD1201_FW_TIMEOUT	(1000)
 
-#define ZD1201_TX_TIMEOUT	(2*HZ)
+#define ZD1201_TX_TIMEOUT	(2000)
 
 #define ZD1201_USB_CMDREQ	0
 #define ZD1201_USB_RESREQ	1
